README.md

```diff
@@ -9,23 +9,16 @@ This is a simple github action that is triggered on github PR merging that will
 
 To use this, simply define an action configuration yaml in the `.github/workflows` directory of your repository with the following contents:
 ```
-name: auto_tagger
+name: Auto Tagger
+
 on:
   pull_request:
-   types: [closed]
+    types: [closed]
+
 jobs:
-  build:
-    if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true
-    runs-on: ubuntu-20.04
-    steps:
-    - uses: actions/checkout@master
-      with:
-        fetch-depth: 0
-    - name: Bump version and push tag
-      uses: RueLaLa/auto-tagger@master
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        GITHUB_PR_NUMBER: ${{ github.event.number }}
+  auto-tagger:
+    uses: RueLaLa/auto-tagger/.github/workflows/auto_tagger.yml@master
```

Comment:
So this functions, but I was a little hasty with [the last auto-tagger PR](https://github.com/RueLaLa/auto-tagger/pull/10). I think we should relocate the file used by other repos so we can decouple the auto-tagger repo's internal config from the functioning of other repos.